src/index.js

```diff
@@ -358,7 +358,7 @@ async function fetchComponents(fileId, componentNames, pageId = '', pageName = '
       // Check if any requested components were not found
       const notFound = componentNames.filter(name => !nameMap.has(name));
       if (notFound.length > 0) {
-        console.log(chalk.yellow(`Warning: The following components were not found: ${notFound.join(', ')}`));
+        console.log(chalk.red(`Warning: The following components were not found: ${notFound.join(', ')}`));
```

Comment:
If we change this to red, should it be treated as an error?

Reply:
No, it just stands out.

![Screenshot 2025-05-15 at 3 54 10 PM](https://github.com/user-attachments/assets/0c91195c-5945-4979-9f0d-759144a0ba17)

Reply:
Warning: to me implies a yellow color, not red.